mlrun/feature_store/retrieval/spark_merger.py

```diff
@@ -46,12 +47,12 @@ def spark_df_to_pandas(spark_df):
                 )
                 type_conversion_dict[field.name] = "datetime64[ns]"
 
-        df = PandasConversionMixin.toPandas(spark_df)
+        df = toPandas(spark_df)
```

Comment:
It's a bit confusing that some places use the spark_df_to_pandas function for special datetime handling, while others use toPandas directly without it.

We should decide whether we want to include datetime handling and consistently apply this choice wherever needed.